frepple/scripts/xml/frepple_6.11.0.xsd

```diff
@@ -1259,18 +1148,17 @@
       <xsd:attribute name="source" type="xsd:normalizedString" />
       <xsd:attribute name="current" type="xsd:dateTime" />
       <xsd:attribute name="logfile" type="xsd:normalizedString" />
-      <xsd:attribute name="plannable" type="xsd:boolean" />
       <xsd:attribute name="detectproblems" type="xsd:boolean" />
+      <xsd:attribute name="individualPoolResources" type="xsd:boolean" />
     </xsd:complexType>
   </xsd:element>
 
   <!--
     Frepple also recognizes the following XML processing instructions.
     Note that these instructions are never validated or checked by the parser.
     They are only processed inside frepple.
-
       <?python xxxx ?>
       Include Python code in your model.
   -->
 
-</xsd:schema>
+</xsd:schema>
```

Comment:
missing new line at the end of the file.
This XSD is quite big isn't it?

Reply:
As provided :) thanks